common/ch_channel_map/bionic

```diff
@@ -4,6 +4,7 @@
 # Versions are based on https://docs.openstack.org/charm-guide/latest/project/charm-delivery.html
 
 CHARM_CHANNEL[vault]=1.6/edge
+CHARM_CHANNEL[barbican-vault]=latest/edge
```

Comment:
Did you find documentation on what channel we should be using here?

Reply:
Unfortunately, I have not found any documentation that points to any specific mentions of channel for barbican-vault should be used for which OpenStack version / Ubuntu series other than it's only supported on Openstack Rocky and above.

FWIW, the [barbican-vaultcharm](https://charmhub.io/barbican-vault) only has two related OpenStack version channels there: 
- `yoga-stable`
- `xena/stable`

Should I adjust those on this PR?

The only other two channels are `latest/stable` and `latest/edge`.

Reply:
Let's see what the product team thinks. Could you ask them and find out what channel we should be using for what version of OS / OpenStack?

Reply:
@nicolasbock I've force pushed a new commit after checking with the OS team what are the right channel versions to use.